modules/sdk-core/src/bitgo/utils/opengpgUtils.ts

```diff
@@ -132,13 +132,13 @@ export async function verifySharedDataProof(
   const senderPubKey = await pgp.readKey({ armoredKey: senderPubKeyArm });
   const signedKey = await pgp.readKey({ armoredKey: keyWithNotation });
   if (
-    !(await verifyPrimaryUserWrapper(signedKey, senderPubKey, true).then((values) =>
+    !(await verifyPrimaryUserWrapper(signedKey, senderPubKey, false).then((values) =>
       _.some(values, (value) => value.valid)
     ))
   ) {
     return false;
   }
-  const primaryUser = await signedKey.getPrimaryUser();
+  const primaryUser = await signedKey.getPrimaryUser(null as unknown as undefined);
```

Comment:
what a line... 

we should create a type def for this to override their invalid types. can do this in a follow up.

Reply:
yeah agreed, almost puked while writing this